woocommerce/woocommerce-admin
# #3793 Tweak: Create admin note if Jetpack plugin doesn't get installed due to an error during OBW

Fixes #3793

This creates an admin note prompting the user to retry install of the Jetpack or WooCommercev Services plugins if they failed on installation during the wizard.

### Detailed test instructions:

1. Delete the Jetpack plugin if it is installed
2. Simulate a failing plugin install by adding `true ||` to the condition on line ~241~ 253 of `OnboardingPlugins.php`:
![image](https://user-images.githubusercontent.com/224531/76587245-ff0a7d80-652e-11ea-8bac-67424ea98e18.png)
3. Delete all existing notes from the `wp_wc_admin_notes` table
4. Activate the onboarding wizard via WooCommerce -> Reports -> Help -> Setup wizard -> Profile Setup Wizard -> Enable
5. Proceed through the wizard, make sure you choose to install Jetpack
6. When Jetpack installation fails, navigate to `/wp-admin/admin.php?page=wc-reports` to break out of the onboarding wizard.
7. open the inbox and open the new message
8. Click the "Install plugins" CTA
9. The button will go inactive for a while then the page will reload
10. The message will have been actioned (won't be shown)
11. The Jetpack and WooCommerce Services plugins should be installed and activated.

The notification should appear in the inbox as follows:

![image](https://user-images.githubusercontent.com/224531/76587342-44c74600-652f-11ea-870a-5c00b9f7f66f.png)

### Changelog Note:

Tweak: Create admin note if Jetpack or WooCommerce Services plugin doesn't get installed due to an error during OBW
